### PR TITLE
Adam Sturge - HW 00

### PR DESCRIPTION
**Pull request by AdamSturge**

### CODE REVIEW

**Comment by alecjacobson** [comment]
in future projects I will try to add `build/` to the .gitignore, but in case I forget. Please try not to commit this (it's big with binaries etc.)

File: src/edges.cpp [hunk 1]
```diff
@@ -1,8 +1,55 @@
...
-  return E;
+    // O(|F|log|F|)
+	// Use additional O(|F|) storage to create edges set. 
+    // std::set supports O(log(|F|)) insert and lookup. 
+	// NOTE: Can change to unordered_set to support O(1) lookup if I provide hash function for pair<int,int>
+	Eigen::MatrixXi E;
+	Eigen::MatrixXi EBig(3*F.rows(), 2);
+	std::set<std::pair<int, int>> m;
+	int lostRows = 0;
+	std::pair<int, bool> p;
```
Comment (alecjacobson) [inline src/edges.cpp]: Seems like this has a typo. The code will work if this line is replaced with

    std::pair<int, int> p;

Comment (AdamSturge) [inline src/edges.cpp]: Yes that is a typo.

**Comment by alecjacobson** [comment]
✔️

**Pull request closed without merging**